app/controllers/lists_controller.rb

```diff
@@ -9,13 +9,14 @@ class ListsController < ApplicationController
   # POST /lists
   #----------------------------------------------------------------------------
   def create
-    list_params[:user_id] = (current_user.id if params[:is_global].to_i.zero?)
+    list_attr = list_params.to_h
+    list_attr[:user_id] = (current_user.id if params[:is_global].t o_i.zero?)
```

Comment:
```suggestion
    list_attr[:user_id] = (current_user.id if params[:is_global].to_i.zero?)
```

Reply:
Sorry my bad, I must have put a white space there by mistake, let me just amend it.